ee/tabby-webserver/src/schema.rs

```diff
@@ -15,9 +15,21 @@ impl Query {
     }
 }
 
-pub type Schema =
-    RootNode<'static, Query, EmptyMutation<ServerContext>, EmptySubscription<ServerContext>>;
+#[derive(Default)]
+pub struct Mutation;
+
+#[graphql_object(context = ServerContext)]
+impl Mutation {
+    async fn refresh_token(ctx: &ServerContext, old: String, new: String) -> String {
```

Comment:
new token should be generated by server directly. There's no need to pass in old token